pkg/cloud/rgraph/rnode/backendservice/node.go

```diff
@@ -69,9 +69,9 @@ func (n *backendServiceNode) Diff(gotNode rnode.Node) (*rnode.PlanDetails, error
 	}
 
 	for _, delta := range diff.Items {
-		//TODO(kl52752) explore which fields can't be updated
 		switch {
-		case delta.Path.Equal(api.Path{}.Pointer().Field("LoadBalancingScheme")):
+		case delta.Path.Equal(api.Path{}.Pointer().Field("LoadBalancingScheme")),
```

Comment:
Let's add a brief comment here to help future maintainers reason about the behavior:

```
// These fields cannot be changed in place and require the
// resource to be recreated.
```

Reply:
done